index.html

```diff
@@ -9,18 +9,25 @@ <h1>Akshay Sharma</h1>
         <section id="about">
         <h2>About</h2>
    
-            <p>My name is Akshay. I am very excited to be part of Code the Dream. I am very Impressed with the people involved and the structure of this course.  </p>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga maiores facilis repudiandae ea sed officia dolor hic reprehenderit itaque tempore!</p>
-        </section>
+                <p>My name is Akshay. I am very excited to be part of Code the Dream. I am very Impressed with the people involved and the structure of this course.  </p>
+                
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga maiores facilis repudiandae ea sed officia dolor hic reprehenderit itaque tempore!</p>
+        </section id="experience">
```

Comment:
the id attribute should be in the defining or HTML tag, it should not be the closing tag